docs/sql.rst

```diff
@@ -513,7 +513,7 @@ Technically you can also specify expression as a $table property of your model::
         function init(): void {
             parent::init();
 
-            $this->init = $this->expr("call get_report_data()");
+            $this->table = $this->expr('call get_report_data()');
```

Comment:
table can be an expr?

Reply:
I have not tested it now but this is in original documentation and I don't see why not as it is handled by dsql template in this case. So [table] in the template is replaced with the expression